internal/upload/koji/koji.go

```diff
@@ -162,7 +162,7 @@ func NewFromPlain(server, user, password string, transport http.RoundTripper) (*
 	// The API doesn't require sessionID, sessionKey and callnum yet,
 	// so there's no need to use the custom Koji RoundTripper,
 	// let's just use the one that the called passed in.
```

Comment:
Is this comment still up-to-date?

Reply:
I believe it is, I will double check it.